float32 front_left_angle 	# 0 to 180 angle in degrees where 180 is fully extended (pushing down), 0 is fully bent (up in the air)
float32 front_right_angle
float32 back_left_angle
float32 back_right_angle